clients/java/src/main/java/io/camunda/zeebe/client/api/search/query/DecisionDefinitionQuery.java

```diff
@@ -15,13 +15,13 @@
  */
 package io.camunda.zeebe.client.api.search.query;
 
+import io.camunda.client.api.search.request.DecisionDefinitionSearchRequest;
 import io.camunda.zeebe.client.api.search.filter.DecisionDefinitionFilter;
 import io.camunda.zeebe.client.api.search.response.DecisionDefinition;
 import io.camunda.zeebe.client.api.search.sort.DecisionDefinitionSort;
 
 /**
- * @deprecated since 8.8 for removal in 8.9, replaced by {@link
- *     io.camunda.client.api.search.query.DecisionDefinitionQuery}
+ * @deprecated since 8.8 for removal in 8.9, replaced by {@link DecisionDefinitionSearchRequest}
```

Comment:
```suggestion
 * @deprecated since 8.8 for removal in 8.9, replaced by {@link io.camunda.client.api.search.request.DecisionDefinitionSearchRequest}
```
I prefer not mixing the imports with different namespaces

Reply:
Good catch, totally agree, classic IDE auto-adjustment. Changed with https://github.com/camunda/camunda/pull/30219/commits/530eab88dd90ea689c11aa4481c3c9cbf5aa33b2.